lambdas/service/app.py

```diff
@@ -406,11 +406,17 @@ def get_integrations():
     try:
         entity_ids = query_params['entity_ids']
     except KeyError:
+        # Case where parameter is absent (any id okay)
```

Comment:
Unsure what is meant here by "any id okay". Could you elaborate or remove if not needed?

Reply:
Would these updated comments be sufficiently clear?

409        # Case where parameter is absent (do not filter by id)
413        # Case where parameter is present and has a value (filter for matching id value)
416        # Case where parameter is present but lacks a value (filter for missing entity_id field, i.e., there are no acceptable id values)

Reply:
These comment messages look good, thanks!